tests/e2e/contracts/llm_erc20.py

```
@@ -0,0 +1,69 @@
+            gl.eq_principle_prompt_non_comparative(
+                lambda: input,
+                task=task,
+                criteria=criteria,
+            )
+            .replace("```json", "")
+            .replace("```", "")
+        )
+        print("final_result: ", final_result)
+        result_json = json.loads(final_result)
+        for k, v in result_json["updated_balances"].items():
+            self.balances[Address(k)] = v
```

Ruff (0.11.9)

`Address` may be undefined, or defined from star imports

(F405)